### PR TITLE
Return `cdc.Event` instead of `any`

### PR DESCRIPTION
**Pull request by Tang8330**
This will return the payload in the Artie Event format which will make it easier for us to do subsequent saves.

### CODE REVIEW

File: lib/types.go [hunk 1]
```diff
@@ -18,20 +19,20 @@ func NewRawMessage(topicSuffix string, partitionKey map[string]any, payload util
...
 		TopicSuffix:  topicSuffix,
 		PartitionKey: partitionKey,
-		mongoPayload: payload,
+		mongoPayload: &payload,
 		mongo:        true,
 	}
 }
 
-func (r RawMessage) GetPayload() any {
+func (r RawMessage) GetPayload() cdc.Event {
```
Comment (nathan-artie) [inline lib/types.go:35]: Can we just make `payload` a `cdc.Event` to begin with, then we don't have to have `both `payload` and `mongoPayload` as well as `mongo`?

Comment (Tang8330) [reply lib/types.go:31]: Yes we can, nice!

File: lib/types.go [hunk 2]
```diff
@@ -1,40 +1,33 @@
...
 )
 
 type RawMessage struct {
 	TopicSuffix  string
 	PartitionKey map[string]any
-	payload      util.SchemaEventPayload
-	mongoPayload mongo.SchemaEventPayload
-
-	mongo bool
+	payload      cdc.Event
```
Comment (nathan-artie) [inline lib/types.go:12]: Nit: Can we call this `event`?

File: lib/types.go [hunk 3]
```diff
@@ -1,40 +1,33 @@
...
 }
 
 func NewRawMessage(topicSuffix string, partitionKey map[string]any, payload util.SchemaEventPayload) RawMessage {
```
Comment (nathan-artie) [inline lib/types.go:15]: Can `NewRawMessage` just accept `cdc.Event` and then we don't need `NewMongoMessage`?

File: lib/types.go (hunk 2 above)
Comment (nathan-artie) [reply lib/types.go]: Should we add getters on `TopicSuffix` and `PartitionKey` too then?

Comment (nathan-artie) [reply lib/types.go]: Could drop the `Get` and call it `Event` since `Get*` to me feels like there might be underlying logic.

Comment (Tang8330) [reply lib/types.go]: Sure

**Review by nathan-artie — APPROVED** [review]

File: integration_tests/utils/utils.go [hunk 4]
```diff
@@ -44,11 +44,11 @@ func ReadTable(db *sql.DB, dbzAdapter transformer.Adapter) ([]lib.RawMessage, er
 }
 
 func GetPayload(message lib.RawMessage) util.SchemaEventPayload {
-	payloadTyped, ok := message.GetPayload().(util.SchemaEventPayload)
+	payloadTyped, ok := message.GetEvent().(*util.SchemaEventPayload)
 	if !ok {
-		panic("payload is not of type util.SchemaEventPayload")
+		panic("payload is not of type *util.SchemaEventPayload")
```
Comment (nathan-artie) [inline integration_tests/utils/utils.go:49]: Nit: `event is not of type`.

**Pull request merged**